### PR TITLE
Patterncommands: Framework compatibility update

### PR DESCRIPTION
**Pull request by feralheart**

### CODE REVIEW

File: packages/pattern-commands/src/lib/structures/PatternCommand.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Args, Command } from '@sapphire/framework';
+import { Args, Command, MessageCommand } from '@sapphire/framework';
```
Comment (c43721) [inline packages/pattern-commands/src/lib/structures/PatternCommand.ts:1]: `Command` is not used

File: packages/pattern-commands/src/lib/structures/PatternCommand.ts [hunk 2]
```diff
@@ -36,7 +36,7 @@ export namespace PatternCommand {
 	export type Context = Command.Context;
 
 	/** Re-export of {@link Command.RunContext} */
```
Comment (c43721) [inline packages/pattern-commands/src/lib/structures/PatternCommand.ts:38]:
```diff
- /** Re-export of {@link Command.RunContext} */
+ /** Re-export of {@link MessageCommand.RunContext} */
```

**Review by c43721 — CHANGES_REQUESTED** [review]

File: packages/pattern-commands/src/lib/structures/PatternCommand.ts (hunk 1 above)
Comment (feralheart) [reply packages/pattern-commands/src/lib/structures/PatternCommand.ts:1]: `PatternCommand` class is extended from `Command`

**Comment by vladfrangu** [comment]
Before we can merge this you'll need to also update lock files/package.json!

**Comment by feralheart** [comment]
@vladfrangu Just the ones in the plugin dir, or the main ones as well?

**Comment by vladfrangu** [comment]
For this specific plugin

**Comment by feralheart** [comment]
Done :)

**Comment by vladfrangu** [comment]
You need to update the framework version..

**Comment by feralheart** [comment]
@vladfrangu I've added the main ones as well

**Comment by vladfrangu** [comment]
Ugh... either way we are getting CI errors... Can you merge your PR into #271? Feel free to discuss on Discord

**Comment by feralheart** [comment]
I've made the PR to @c43721 's fork. Hope it'll fix the CI errors

**Comment by c43721** [comment]
yay more build errors

**Comment by c43721** [comment]
@vladfrangu we can close this in favor of #271

**Comment by feralheart** [comment]
Closing in favor of #271

**Pull request closed without merging**